userspace/engine/falco_engine_version.h

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 
 // The version of this Falco engine
 #define FALCO_ENGINE_VERSION_MAJOR 0
-#define FALCO_ENGINE_VERSION_MINOR 46
+#define FALCO_ENGINE_VERSION_MINOR 47
```

Comment:
Should we bump the major? :thinking: 
cc @jasondellaluce

Reply:
I agree with bumping the major

Reply:
No problem for me; if we all agree, i'll push the change.
cc @jasondellaluce

Reply:
/hold while we wait for response.

Reply:
> We never bumped the major, but we did remove fields in the past 🤔 should be ok with minor I think

That's precisely why I'm asking for this. 
I'm okay with bumping the minor since, even for semver, backward incompatible changes can be introduced at any time when the major is `0`. Just wanted to double-check.

Reply:
Briefly discussed with other maintainers, let's keep the minor bump for now.